### DISCUSSION TITLE
loosing path to location to save media

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#276] [General]
when you click accidentally in the box where the path is saved to save media
![image](https://user-images.githubusercontent.com/39587825/166568113-70365ae9-b526-4b0a-93c5-2c899e321aa1.png)
you get the windows explorer
but when you cancel
the path is gone

of course, one can click on select instead of cancel
not mission-critical ;-)

**Comment by sircharlo** [comment]
Just for you: https://github.com/sircharlo/jw-meeting-media-fetcher/commit/44a5fc0642e48cc2e8c371f75935114f92c79adf

![image](https://user-images.githubusercontent.com/1275666/166571090-f16dd15c-f426-49b5-9ce2-13764ec6c351.png)

Will be in next point release 😉

**Comment by riemsesy** [comment]
🫂
